app/post.go

```diff
@@ -673,7 +670,7 @@ func (a *App) UpdatePost(c *request.Context, post *model.Post, safeUpdate bool)
 		a.Srv().Go(func() {
 			pluginContext := pluginContext(c)
 			pluginsEnvironment.RunMultiPluginHook(func(hooks plugin.Hooks) bool {
-				hooks.MessageHasBeenUpdated(pluginContext, newPost, oldPost)
+				hooks.MessageHasBeenUpdated(pluginContext, newPost.ForPlugin(), oldPost.ForPlugin())
```

Comment:
Shouldn't we make similar changes here?

Reply:
Agree.

Reply:
@ashishbhate yes, missed it, thx! Fixed now.